src/Column/MoneyColumn.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/*
+ * Symfony DataTables Bundle
+ * (c) Omines Internetbureau B.V. - https://omines.nl/
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Omines\DataTablesBundle\Column;
+
+use Omines\DataTablesBundle\DataTable;
+use Omines\DataTablesBundle\Exception\MissingDependencyException;
+use Symfony\Component\Form\Extension\Core\DataTransformer\MoneyToLocalizedStringTransformer;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * MoneyColumn.
+ *
+ * @author Max Mishyn <max.mishyn@wisetiger.co.uk>
+ */
+class MoneyColumn extends AbstractColumn
+{
+    /**
+     * @var MoneyToLocalizedStringTransformer
+     */
+    private $transformer;
+
+    public function normalize($value): string
+    {
+        return $this->isRaw() ? (string) $value : $this->format(intval($value));
+    }
+
+    protected function configureOptions(OptionsResolver $resolver)
+    {
+        parent::configureOptions($resolver);
+
+        $resolver
+            ->setDefaults([
+                'raw' => false,
+                'currency' => '',
+                'divisor' => 1,
+                'scale' => 2,
+            ])
+            ->setAllowedTypes('raw', 'bool')
+            ->setAllowedTypes('currency', 'string')
+            ->setAllowedTypes('divisor', 'int')
+            ->setAllowedTypes('scale', 'int')
+        ;
+
+        return $this;
+    }
+
+    public function initialize(string $name, int $index, array $options, DataTable $dataTable)
+    {
+        parent::initialize($name, $index, $options, $dataTable);
+        if (class_exists('Symfony\Component\Form\Extension\Core\DataTransformer\MoneyToLocalizedStringTransformer')) {
+            $this->transformer = new MoneyToLocalizedStringTransformer($this->options['scale'], true, \NumberFormatter::ROUND_HALFUP, $this->options['divisor']);
+        } else {
+            throw new MissingDependencyException('You must have Symfony\Form installed to use ' . self::class);
+        }
+    }
+
+    public function isRaw(): bool
+    {
+        return $this->options['raw'];
+    }
+
+    public function getCurrency(): ?string
+    {
+        return $this->options['currency'];
+    }
+
+    public function getDivisor(): ?int
+    {
+        return $this->options['divisor'];
+    }
+
+    public function getScale(): ?int
+    {
+        return $this->options['scale'];
+    }
+
+    public function isValidForSearch($value): bool
+    {
+        return is_numeric($value);
+    }
+
+    public function format(int $val): string
+    {
+        if (empty($this->getCurrency())) {
+            return $this->transformer->transform($val);
+        }
+
+        $locale = \Locale::getDefault();
+        $format = new \NumberFormatter($locale, \NumberFormatter::CURRENCY);
+        $pattern = $format->formatCurrency(123, $this->getCurrency());
```

Comment:
What is this 'magic' number doing here, and what does the regexp do with it afterwards?

Reply:
Code on this line formats number 123 (just a dummy number, we will not use it later) with requested locale setting and currency.  As a result we receive i.e. "€123.00" for "EUR" currency. Regexp further determines currency symbol ("€" and not "EUR") and it's position (left or right from the number itself) and appends/prepends it to the transformed string

Reply:
That seems like a fragile kludge, and one that is tailored to Western conventions and not entirely future proof. I'm not really familiar with Intl's currency formatting, but shouldn't its whole purpose be that we do not need fake formatters and regular expressions to uniformly implement this behavior?

Reply:
As I explained in another comment, intl does not have dividor setting, which is important in money formatting.  This code simply "injects" those divider and scale support into intl's number/currency formatting. 
In general there are just 3 options for formatting number with a currency code symbol: to the left from the number, to the right from the number and just a number without currency symbol. Two first are covered with this conversion and regexp trick, the third one, without currency symbol is done whith a condition in 94-96 lines. 
I'm quite sure that it is not Western only solution, as it works in one of my projects in Hebrew language (which is RtL itself) with israeli shekels without any problems and formats like "₪561.20". 
I can add some Eastern and currencies/locales with both LTR and RTL languages to the tests just to provide more peace of mind.